src/features/contactmoment/ContactmomentVragenMenu.vue

```diff
@@ -0,0 +1,97 @@
+<template>
+  <SimpleDialog
+    :dialog="dialog"
+    message="Let op, je hebt het contactverzoek niet afgerond. Als je deze vraag verlaat, wordt het contactverzoek niet verstuurd."
+  />
+  <menu class="vragen-menu">
+    <li v-for="(vraag, idx) in contactmomentStore.vragen" :key="idx">
+      <span v-if="vraag === contactmomentStore.huidigeVraag">
+        {{ idx + 1 }}
+      </span>
+      <button
+        v-else
+        type="button"
+        :title="`Ga naar vraag ${idx + 1}`"
+        @click="switchVraag(vraag)"
+      >
+        {{ idx + 1 }}
+      </button>
+    </li>
+    <li>
+      <button
+        class="icon-after plus new-question"
+        type="button"
+        title="Nieuwe vraag"
+        @click="startNieuweVraag"
+      ></button>
+    </li>
+  </menu>
+</template>
+<script lang="ts" setup>
+import { useContactmomentStore, type Vraag } from "@/stores/contactmoment";
+import { useConfirmDialog } from "@vueuse/core";
+import SimpleDialog from "../../components/SimpleDialog.vue";
+
+const contactmomentStore = useContactmomentStore();
+const dialog = useConfirmDialog();
+
+async function startNieuweVraag() {
+  if (await validateContactverzoek()) {
+    contactmomentStore.startNieuweVraag();
+  }
+}
+
+async function switchVraag(vraag: Vraag) {
+  if (await validateContactverzoek()) {
+    contactmomentStore.huidigeVraag = vraag;
+  }
+}
+
+async function validateContactverzoek() {
+  if (!contactmomentStore.huidigeVraag.contactverzoek.isDirty) return true;
```

Comment:
I've never really worked with a boolean checking if something is dirty, and I'm unsure why we're doing it. If you have some time when you read this, maybe you could explain it to me 😄

Reply:
Dirty is usually used to indicate that a form input has entered and left a focus state. A common use for this is to make validation messages less intrusive, by hiding them until somebody has actually tried to fill in a field.

In this case, I think it's confusing to use this term. What we're actually interested in, is if the user was in the progress of filling in the form. Which means one or more fields not only have to be 'dirty', but also filled in. I will see what I can do to clarify this. Maybe the term isInProgress is better here.

Reply:
Changed it to isInProgress